agenta-web/src/components/pages/evaluations/evaluationResults/EvaluationResults.tsx

```diff
@@ -327,7 +328,8 @@ const EvaluationResults: React.FC<Props> = () => {
                                         )
                                     )
                                         return
-                                    EvaluationStatus.FINISHED === params.data?.status &&
+                                    ;(params.data?.status === EvaluationStatus.FINISHED ||
```

Comment:
```suggestion
                                    (params.data?.status === EvaluationStatus.FINISHED ||
```

I think the semi-colon (;) will cause the frontend to break.